docs/usage-as-operator.md

```diff
@@ -51,6 +51,8 @@ spec:
     class: standard
   - name: pd-ssd
     class: premium
+  - name: SCRATCH
```

Comment:
Also, a short description above this YAML example would be helpful to explain the special `SCRATCH` volume type.

Reply:
Accept